commit 2ad2d44e9391cdd4e8c980d8544b2b0f45eabb35
Author: Rin Kuryloski <kuryloskip@vmware.com>
Date:   Tue Jan 17 14:58:41 2023 +0100

    Avoid unnecessary use_repo repos in MODULE.bazel
    
    Not all the listed deps need be declared here

diff --git a/MODULE.bazel b/MODULE.bazel
index f63ac05..2d62751 100644
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -23,7 +23,6 @@ bazel_dep(
     version = "0.28.0",
 )
 
-# The union of go deps from bazel-gazelle and this repo
 go_deps = use_extension("@bazel_gazelle//:extensions.bzl", "go_deps")
 
 go_deps.from_file(go_mod = "@rules_erlang//:go.mod")
@@ -31,18 +30,10 @@ go_deps.from_file(go_mod = "@rules_erlang//:go.mod")
 use_repo(
     go_deps,
     "com_github_bazelbuild_buildtools",
-    "com_github_bmatcuk_doublestar_v4",
-    "com_github_fsnotify_fsnotify",
     "com_github_emirpasic_gods",
+    "com_github_ghodss_yaml",
     "com_github_google_go_cmp",
-    "com_github_pelletier_go_toml",
-    "com_github_pmezard_go_difflib",
     "in_gopkg_yaml_v2",
-    "org_golang_x_mod",
-    "org_golang_x_sync",
-    "org_golang_x_tools",
-    # Used internally by the go_deps module extension.
-    "bazel_gazelle_go_repository_directives",
 )
 
 erlang_package = use_extension(
